Sources/XMTPTestHelpers/TestHelpers.swift

```diff
@@ -264,15 +259,19 @@ public struct Fixtures {
 
 	public var bob: PrivateKey!
 	public var bobClient: Client!
+	let clientOptions: ClientOptions? = ClientOptions(
+		api: ClientOptions.Api(env: XMTPEnvironment.local, isSecure: false)
+	)
 
 	init() async throws {
 		alice = try PrivateKey.generate()
 		bob = try PrivateKey.generate()
 
 		fakeApiClient = FakeApiClient()
```

Comment:
I really want to remove all the fakeApiClient stuff from here too. A lot of tests aren't really testing what we think they are.